### PR TITLE
[BPK-3348] Screenshot automation with Fastlane

### PR DESCRIPTION
**Pull request by georgegillams**
Try it!
`rake take_screenshots`

### CODE REVIEW

File: Example/Backpack/View Controllers/ToastViewController.swift [hunk 1]
```diff
@@ -44,14 +44,14 @@ class ToastViewController: UIViewController {
         toast.mode = .indeterminate
         toast.labelText = "This is the title"
         toast.detailsLabelText = "Details label text"
-        toast.hide(true, afterDelay: 1.2)
+        toast.hide(true, afterDelay: 5)
```
Comment (georgegillams) [inline Example/Backpack/View Controllers/ToastViewController.swift]: I had to change these to ensure they are visible when the screenshot is captured

File: Rakefile [hunk 2]
```diff
@@ -103,6 +105,16 @@ task :lint do
   sh "bundle exec pod lib lint --allow-warnings"
 end
 
+task :take_screenshots do
+  # Remove existing screenshots
+  FileUtils.rm_rf(Dir.glob('screenshots/*'))
```
Comment (k0nserv) [inline Rakefile]: Do we have to glob here? `rm_rf` already removes recursively so

```ruby
FileUtils.rm_rf('screenshots')
```

should be enough

Comment (georgegillams) [reply Rakefile]: But we don't want to delete `screenshots`, just everything inside it. Hence the `glob` and the `*`

Comment (k0nserv) [reply Rakefile]: Isn't everything in `screenshots` generated though? Seems cleaner to just remove the whole thing

Comment (georgegillams) [reply Rakefile]: Actually it still works. I guess `fastlane` recreates the directory so 🤷‍♀️

**Pull request merged**